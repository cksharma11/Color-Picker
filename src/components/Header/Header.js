import React from "react";

const Header = () => {
  return <div className="div-header">Color Picker</div>;
};

export default Header;
