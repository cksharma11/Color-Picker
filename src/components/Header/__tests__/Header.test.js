import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import Header from "../Header";

Enzyme.configure({ adapter: new Adapter() });

describe("Header", () => {
  it("should render properly", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
