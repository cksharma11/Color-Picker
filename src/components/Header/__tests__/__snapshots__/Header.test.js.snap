// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render properly 1`] = `
"<div className=\\"div-header\\">
  Color Picker
</div>"
`;
